src/main/webapp/js/finscholar/commonlistview/commonlistview.js

```diff
@@ -134,12 +152,22 @@ class CommonListView extends BasicView {
    * Loads the first two batches of list item to page.
    */
   async renderView() {
-    super.setCurrentContent(commonlistview({pagetype: this.optionTag_}));
+    super.setCurrentContent(commonlistview({
+      pagetype: this.optionTag_,
+      sortParams: SORT_PARAMS_MAP,
+      asc: ASCENDING,
+      desc: DESCENDING}));
     super.resetAndUpdate();
     this.initJsaction_();
     // tableContainer.innerHTML = commonlistview({pagetype: this.optionTag_});
     this.container_ = googDom.getElement(ITEM_CONTAINER_ID);
     this.statusBar_ = googDom.getElement(STATUS_BAR_ID);
+    this.sortBySelector = googDom.getElement(SORT_BY_SELECTOR_ID);
+    this.sortBySelector.value = this.sortBy;
+    this.sortBySelector.addEventListener('change', this.bindedSelectorHandler_);
```

Comment:
Wow I'm surprised that addEventListener works here. I not sure whether it's better than jsaction, so let's first keep this here and make changes if needed later.